### ISSUE TITLE
inconsistency in search_targetpixelfile (some cases)

### ISSUE THREAD

**Issue by jpdeleon** [KeplerGO/lightkurve#533]
#### Problem description

Given coordinates, the result of `search_targetpixelfile` will be reported including TIC ID matches.
When one of TIC ID reported is supplied in the `search_targetpixelfile`, the tpf cannot be searched instead.

#### Example

```python
import lightkurve as lk
from astropy.coordinates import SkyCoord

ra, dec = 47.0935, -60.1725
c = SkyCoord(ra=ra,dec=dec,unit='deg')
res = lk.search_targetpixelfile(c, mission='TESS')

SearchResult containing 2 data products.

target_name                     productFilename                        description     distance
----------- ------------------------------------------------------- ------------------ --------
  220571983 tess2018234235059-s0002-0000000220571983-0121-s_tp.fits Target pixel files      0.0
  220571983 tess2018263035959-s0003-0000000220571983-0123-s_tp.fits Target pixel files      0.0
```
Then if the reported TIC ID is supplied instead:
```
res = lk.search_targetpixelfile(220571983, mission='TESS')
```
No data found for target "220571983".

It appears this only happens for some targets only, e.g. [ra,dec=(80.0694 -51.9997), tic id=219423302] also. Other queries with matching TIC ID and coordinate  should generally work, but this may be a hint of some inherent bug in `search_targetpixelfile` or data source.

#### Environment:
- platform (e.g. Linux, OSX, Windows): Linux
- lightkurve version (e.g. 1.0b6): 1.0.1
- installation method (e.g. pip, conda, source): pip

**Comment by mirca** [comment]
I'm able to reproduce this issue with the current dev version of ``lightkurve``.  Thanks for reporting @jpdeleon !

**Comment by mirca** [comment]
This ``res = lk.search_targetpixelfile('TIC 220571983')`` works though.

**Comment by jpdeleon** [comment]
I checked that adding "TIC" to TIC ID indeed solves the problem.

**Comment by barentsen** [comment]
An underlying issue here is that 220571983 appears both in the EPIC catalog (i.e. the K2 input catalog) and in the TIC catalog (i.e. the TESS input catalog), and "TIC 220571983" is a completely different position compared to "EPIC 220571983".

I wonder if Lightkurve should emit a warning whenever a number is passed without the explicit prefix "EPIC" or "TIC".  Any thoughts/opinions?

/cc @scfleming @mustaric

**Comment by scfleming** [comment]
IMO, lightkurve should not allow a user to query based on an ID *without* a specifier, e.g., "Kepler" or "KIC" or "K2" or "TIC"...we warn people about that on the MAST side too for the same reason.

**Comment by scfleming** [comment]
Or, automatically append an appropriate string based on the specific mission, if you still would like users to supplay a numerical ID only...

**Comment by barentsen** [comment]
Caveat: `lk.search_targetpixelfile(200164267)` currently returns the custom mask that was observed for TRAPPIST-1 in C12, but `lk.search_targetpixelfile("EPIC 200164267")` does not.  We need to investigate.

**Comment by scfleming** [comment]
The issue is that "EPIC 200164267" is being caught by Simbad, and thus the name resolver is defaulting to the coordinates returned by Simbad and not the EPIC catalog at MAST.  We'd have to file a ticket so that if an EPIC ID string is sent, it will NOT try to do a Simbad/NED lookup.

**Comment by scfleming** [comment]
Otherwise, use of a larger search radius might catch it with the Simbad returned coordinates.

**Comment by barentsen** [comment]
@scfleming Correct!  `search_targetpixelfile("EPIC 200164267", radius=10)` does return the TPF with ID 200164267.

**Comment by gully** [comment]
The core issue here is closed by #558, and the issue of TRAPPIST-1's custom aperture getting missed is a duplicate of #499.
